src/Jarvis.Addin.Files/FileProvider.cs

```diff
@@ -48,6 +48,11 @@ public FileProvider(IFileIndex index, IconLoader loader)
             return await _index.Find(query.Raw?.Trim(), CancellationToken.None).ConfigureAwait(false);
         }
 
+        protected override Task<IQueryResult> CreateFallbackResult(string query)
+        {
+            throw new NotImplementedException();
```

Comment:
Don't think this should be here

Reply:
Yes, this is a bit temporary (forgot to add a comment about that). Any suggestions on how to handle the cases where a provider is not global provider